lib/getShows.js

```diff
@@ -37,6 +39,26 @@ const loadShows = async () => {
     .reverse();
 };
 
+const indexShows= async () => {
+  const { ALGOLIA_APP_ID, ALGOLIA_API_KEY, ALGOLIA_INDEX_NAME } = process.env;
```

Comment:
Good call! Could you possibly do this in `src/components/showList.js` as well? It's something I completely forgot to do.